sonar-ws/src/main/java/org/sonarqube/ws/client/setting/SetRequest.java

```diff
@@ -103,7 +102,7 @@ public Builder setValues(List<String> values) {
       return this;
     }
 
-    public Builder setFieldValues(List<Map<String, String>> fieldValues) {
+    public Builder setFieldValues(List<String> fieldValues) {
```

Comment:
as discussed offline, I've tried it but it brought dependencies in sonar-ws we don't want